Bug 1382204 - move age,size out of internal config

petemoore opened the pull request:
See [bug 1382204](https://bugzilla.mozilla.org/show_bug.cgi?id=1382204) for more details.

At the same time, I've added a travis config file, so someone with admin access (I don't have that) will need to go to https://travis-ci.org/profile and flip the switch on `mozilla/tc-coalesce` repo, ideally before merging this PR, so that the travis builds run.

dividehex commented:
I've flipped the switch and setup travis for this repo.  Also since heroku auto-deploys from the Master branch, I've enabled 'wait for CI'.  We should be good to merge this pending @djmitche 's review. Thanks Pete!

petemoore commented:
@dividehex I've squashed the commits, and also made a minor tweak (declaring `age` and `size` as type `int`) and added tests that we get a `404` HTTP response code if non-integers are passed through, or any other invalid URL. Sorry for the late change!

I'm afraid I don't have merge permission in this repo, so if you are happy with my latest additions, could you directly merge this PR?

Many thanks!

dividehex commented:
Squashed updates look good.  I'll merge and deploy.
